### DISCUSSION TITLE
Change inputType of MessageInputView for better mentioning experience

### DISCUSSION THREAD

**Discussion by rupinderjeet** [GetStream/stream-chat-android#1715] [Ideas]
We can configure the keyboard input to show "@" symbol using something like `textEmailAddress` as `inputType` of MessageInputView. This can help quickly mentioning other users when they are chatting in a channel.

Using `inputType` like this seems a bit wrong because it wouldn't technically be an email.

I don't know much about UX, so, just a suggestion. What are your thoughts?

**Comment by zsmb13** [comment]
As you said, setting the input type to email would break the semantics of the input field. For example, it could affect things like autocompletion now offering to input an email address every time. Mentions are not an extremely frequently used feature, and they're probably accessible enough as-is for now.
